test/integration/001_simple_copy_test/seeds-merge-cols-update/seed.csv

```diff
@@ -0,0 +1,204 @@
+2021-03-05,62,Andrew,Lawson,alawson1p@over-blog.com,Male,54.98.36.94
+2021-03-05,63,Peter,Morgan,pmorgan1q@rambler.ru,Male,14.77.29.106
+2021-03-05,64,Nicole,Garrett,ngarrett1r@zimbio.com,Female,21.127.74.68
+2021-03-05,65,Joshua,Kim,jkim1s@edublogs.org,Male,57.255.207.41
+2021-03-05,66,Ralph,Roberts,rroberts1t@people.com.cn,Male,222.143.131.109
+2021-03-05,67,George,Montgomery,gmontgomery1u@smugmug.com,Male,76.75.111.77
+2021-03-05,68,Gerald,Alvarez,galvarez1v@flavors.me,Male,58.157.186.194
+2021-03-05,69,Donald,Olson,dolson1w@whitehouse.gov,Male,69.65.74.135
+2021-03-05,70,Carlos,Morgan,cmorgan1x@pbs.org,Male,96.20.140.87
+2021-03-05,71,Aaron,Stanley,astanley1y@webnode.com,Male,163.119.217.44
+2021-03-05,72,Virginia,Long,vlong1z@spiegel.de,Female,204.150.194.182
+2021-03-05,73,Robert,Berry,rberry20@tripadvisor.com,Male,104.19.48.241
+2021-03-05,74,Antonio,Brooks,abrooks21@unesco.org,Male,210.31.7.24
+2021-03-05,75,Ruby,Garcia,rgarcia22@ovh.net,Female,233.218.162.214
+2021-03-05,76,Jack,Hanson,jhanson23@blogtalkradio.com,Male,31.55.46.199
+2021-03-05,77,Kathryn,Nelson,knelson24@walmart.com,Female,14.189.146.41
+2021-03-05,78,Jason,Reed,jreed25@printfriendly.com,Male,141.189.89.255
+2021-03-05,79,George,Coleman,gcoleman26@people.com.cn,Male,81.189.221.144
+2021-03-05,80,Rose,King,rking27@ucoz.com,Female,212.123.168.231
+2021-03-05,81,Johnny,Holmes,jholmes28@boston.com,Male,177.3.93.188
+2021-03-05,82,Katherine,Gilbert,kgilbert29@altervista.org,Female,199.215.169.61
+2021-03-05,83,Joshua,Thomas,jthomas2a@ustream.tv,Male,0.8.205.30
+2021-03-05,84,Julie,Perry,jperry2b@opensource.org,Female,60.116.114.192
+2021-03-05,85,Richard,Perry,rperry2c@oracle.com,Male,181.125.70.232
+2021-03-05,86,Kenneth,Ruiz,kruiz2d@wikimedia.org,Male,189.105.137.109
+2021-03-05,87,Jose,Morgan,jmorgan2e@webnode.com,Male,101.134.215.156
+2021-03-05,88,Donald,Campbell,dcampbell2f@goo.ne.jp,Male,102.120.215.84
+2021-03-05,89,Debra,Collins,dcollins2g@uol.com.br,Female,90.13.153.235
+2021-03-05,90,Jesse,Johnson,jjohnson2h@stumbleupon.com,Male,225.178.125.53
+2021-03-05,91,Elizabeth,Stone,estone2i@histats.com,Female,123.184.126.221
+2021-03-05,92,Angela,Rogers,arogers2j@goodreads.com,Female,98.104.132.187
+2021-03-05,93,Emily,Dixon,edixon2k@mlb.com,Female,39.190.75.57
+2021-03-05,94,Albert,Scott,ascott2l@tinypic.com,Male,40.209.13.189
+2021-03-05,95,Barbara,Peterson,bpeterson2m@ow.ly,Female,75.249.136.180
+2021-03-05,96,Adam,Greene,agreene2n@fastcompany.com,Male,184.173.109.144
+2021-03-05,97,Earl,Sanders,esanders2o@hc360.com,Male,247.34.90.117
+2021-03-05,98,Angela,Brooks,abrooks2p@mtv.com,Female,10.63.249.126
+2021-03-05,99,Harold,Foster,hfoster2q@privacy.gov.au,Male,139.214.40.244
+2021-03-05,100,Carl,Meyer,cmeyer2r@disqus.com,Male,204.117.7.88
+2021-03-06,20,Andrew,Davis,adavisj@reddit.com,Male,9.255.67.25
+2021-03-05,83,Josh,Thomas,jthomas2a@ustream.tv,Male,0.8.205.30
+2021-03-05,92,Angela,Scott,ascott2j@goodreads.com,Female,98.119.208.155
```

Comment:
Did you intend to update these in place? For all three have `load_date = 2021-03-06`? I think it's a bit confusing to duplicate these `id`, so I'd rather you include them only once and "sort" the seed by either `id` _or_ `load_date`.

Reply:
Whoops, I did intend for all three to have `load_date = 2021-03-06`. Good catch! I updated it, so the seed is now sorted by `load_date`.